apiserver/params/caas.go

```diff
@@ -32,6 +32,13 @@ type CAASApplicationProvisioningInfoResults struct {
 	Results []CAASApplicationProvisioningInfo `json:"results"`
 }
 
+// CAASUnitTerminationResult holds result to UnitTerminating call.
+type CAASUnitTerminationResult struct {
+	// Stopping is true if the termination of the unit is temporary.
+	Stopping bool
```

Comment:
Maybe `Stopping` would be better renamed as `WillRestart` or similar.
After all, the containing struct is called `CAASUnitTerminationResult` and the suggested name better reflects the semantics. A dying unit is still stopping and needs to run the stop hook. So the name `Stopping` perhaps doesn't best reflect what's happening.